bugbug/repository.py

```diff
@@ -72,6 +73,7 @@ def _transform(commit):
         else:
             type_ = ext
         obj['types'].add(type_)
+        obj['touched'] = len(stats['touched'])
```

Comment:
`touched` is not the number of files modified, it is the number of lines modified.